packages/debugger/lib/ens/actions/index.js

```diff
@@ -0,0 +1,8 @@
+export const RECORD = "ENS_RECORD";
+export function record(address, name) {
+  return {
+    type: RECORD,
+    address,
+    name //may be null
+  };
+}
```

Comment:
Does this need to be renamed? The asymmetric `null`-ness here suggests that we're specifically dealing with primary name records, so I think this action should reflect that.

Reply:
I mean it doesn't *need* to be renamed, but it could be...